tests/test_merkle.py

```
@@ -0,0 +1,96 @@
+
+
+class TestMerkleTree(unittest.TestCase):
+    def test_empty_transactions(self):
+        root = calculate_merkle_root([])
+        self.assertIsNone(root)
+
+    def test_single_transaction(self):
+        tx = {"sender": "alice", "receiver": "bob", "amount": 10}
+        tree = MerkleTree([tx])
+        root = tree.get_merkle_root()
+        self.assertIsNotNone(root)
```

Ruff (0.15.1)

[warning] Use a regular `assert` instead of unittest-style `assertIsNotNone`

Replace `assertIsNotNone(...)` with `assert ...`

(PT009)